### PR TITLE
enfore more lint rules

### PR DESCRIPTION
**Pull request by jnsdls**
does what it says on the tin

### CODE REVIEW

**Comment by jnsdls** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4754?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4754** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4754?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4751** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4751?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @jnsdls and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by joaquim-verges — APPROVED** [review]

**Comment by MananTank** [comment]
Just FYI, I recently filed a bug in biome about `useSortedClasses` rule - It formats concatenated classes in template strings incorrectly - so need to watch out for that 

https://github.com/biomejs/biome/issues/4041

File: apps/dashboard/src/@/components/ui/ScrollShadow/ScrollShadow.tsx [hunk 1]
```diff
@@ -94,36 +94,36 @@ export function ScrollShadow(props: {
...
-        className={`${styles.scrollShadowBottom} ${styles.scrollShadowY}`}
+        className={`${styles.scrollShadowBottom}${styles.scrollShadowY}`}
         ref={shadowBottomEl}
         style={{
           opacity: "0",
         }}
       />
       <div
-        className={`${styles.scrollShadowLeft} ${styles.scrollShadowX}`}
+        className={`${styles.scrollShadowLeft}${styles.scrollShadowX}`}
```
Comment (MananTank) [inline apps/dashboard/src/@/components/ui/ScrollShadow/ScrollShadow.tsx:112]: Wrong formatting here by biome - breaks the layout

**Review by MananTank — CHANGES_REQUESTED** [review]
Need to fix wrong formatting done by biome on template strings. Honestly I don't feel comfortable using `useSortedClasses` because of this

File: apps/dashboard/src/@/components/ui/ScrollShadow/ScrollShadow.tsx (hunk 1 above)
Comment (MananTank) [reply apps/dashboard/src/@/components/ui/ScrollShadow/ScrollShadow.tsx:112]: Same issue in this file 3 more times

Comment (jnsdls) [reply apps/dashboard/src/@/components/ui/ScrollShadow/ScrollShadow.tsx]: fixed by using `cn` for this an any other case of dynamic classNames using template strings

**Pull request merged**